Makefile

```diff
@@ -10,7 +10,13 @@ RELEASE_COMMIT=$(shell git log --abbrev=8 --pretty=format:'%H' -n 1 $(VERSION))
 RELEASE_SHORT_COMMIT=$(shell git log --abbrev=8 --pretty=format:'%h' -n 1 $(VERSION))
 
 COMMIT=$(shell git log --abbrev=8 --pretty=format:'%H' -n 1)
+COMMIT_DATE=$(shell git log --pretty='format:%cd' --date='format:%Y%m%d' -n 1)
```

Comment:
(commit message) It's not proposed, it's merged already for a while..., anyway not a big deal.

Reply:
Good thing you caught that, because this is what I meant by proposed:

https://github.com/avocado-framework/avocado/pull/3035

But, I had forgot to send those patches as a PR to Avocado.

Reply:
OK :-)